schema/codelists/itemClassificationScheme.csv

```diff
@@ -7,3 +7,6 @@ CPC,Central Product Classification,"The Central Product Classification (CPC) is
 OKDP,Russian National Classification of Products by Economic Activity (1994),"Part of the unified system of classification and coding of technical, economic and social information of the Russian Federation, in use for Russian Procurement information until January 1, 2016. Superseded by OKPD.",http://base.consultant.ru/cons/cgi/online.cgi?req=doc;base=LAW;n=143095;fld=134;from=14052-7;rnd=0.33374630799517035
 OKPD,Russian National Classification of Products by Economic Activity (2014),"Part of the national standardization system of the Russian Federation and based on harmonization with the Statistical Classification of Products by Activity in the European Economic Community, 2008. In use from January 1 2014.",http://base.consultant.ru/cons/cgi/online.cgi?req=doc;base=LAW;n=163703;fld=134;from=170864-15;rnd=0.39699640963226557
 CUCOP,Mexican Single Classifier of Public Procurement,"The Single Classifier of Public Procurement (Clasificador Único de las Contrataciones Públicas) is a national scheme in Mexico for the classification of goods, works, services and leases.",https://sites.google.com/site/cnetcucop/
+NAICS,North American Industry Classification System, "The North American Industry Classification System (NAICS) is the standard used by Federal statistical agencies in classifying business establishments for the purpose of collecting, analyzing, and publishing statistical data related to the U.S. business economy.",https://www.census.gov/eos/www/naics/downloadables/downloadables.html
```

Comment:
The NAICS description from census.gov is a bit US-specific. The 1-2 sentences under "Purpose of NAICS" on this page is more generic: https://www.statcan.gc.ca/eng/subjects/standard/naics/2017/v3/introduction

We can keep the current link though.

Reply:
Also, remove space before first `"`.

Reply:
done